packages/navigator/src/browser/navigator-diff.ts

```diff
@@ -0,0 +1,104 @@
+/********************************************************************************
+ * Copyright (C) 2019 TypeFox and others.
```

Comment:
I'm not sure the copyright here should be TypeFox.

Reply:
Any ideas on what this should be?

Reply:
I assume your company or your name would be sufficient.
@marcdumais-work do you have any thoughts?

Reply:
Assuming that @sauny is not a TypeFox employee, then the copyright should be assigned to whomever is the ultimate owner of this code. This can be an employer or the individual, depending. 

@sauny did you sign the ECA as am individual or as an employee?

Reply:
@sauny please could you update copyrights, the rest we can fix on ourself is you don't have time. This a small nice feature. It would be nice to land it :)